buf.yaml

```
@@ -4,7 +4,37 @@ breaking:
     - temporal/api/schedule/v1
     - temporal/api/update/v1
   use:
-    - PACKAGE
+    - WIRE_JSON
+    - RPC_NO_DELETE
+    - PACKAGE_SERVICE_NO_DELETE
+    - PACKAGE_NO_DELETE
+    - PACKAGE_MESSAGE_NO_DELETE
+    - PACKAGE_ENUM_NO_DELETE
+    - ONEOF_NO_DELETE
+    - MESSAGE_NO_REMOVE_STANDARD_DESCRIPTOR_ACCESSOR
+    - EXTENSION_MESSAGE_NO_DELETE
+    - FILE_SAME_SYNTAX
+    - FILE_SAME_CC_ENABLE_ARENAS
+    - FILE_SAME_PHP_GENERIC_SERVICES
+    - FILE_SAME_PY_GENERIC_SERVICES
+    - FILE_SAME_JAVA_GENERIC_SERVICES
+    - FILE_SAME_CC_GENERIC_SERVICES
+    - FILE_SAME_OPTIMIZE_FOR
+    - FILE_SAME_SWIFT_PREFIX
+    - FILE_SAME_RUBY_PACKAGE
+    - FILE_SAME_PHP_NAMESPACE
+    - FILE_SAME_PHP_METADATA_NAMESPACE
+    - FILE_SAME_PHP_CLASS_PREFIX
+    - FILE_SAME_OBJC_CLASS_PREFIX
+    - FILE_SAME_JAVA_STRING_CHECK_UTF8
+    - FILE_SAME_JAVA_PACKAGE
+    - FILE_SAME_JAVA_OUTER_CLASSNAME
+    - FILE_SAME_JAVA_MULTIPLE_FILES
+    - FILE_SAME_GO_PACKAGE
+    - FILE_SAME_CSHARP_NAMESPACE
+    - FIELD_SAME_TYPE
+    - FIELD_SAME_JSTYPE
+    - FIELD_SAME_CTYPE
```

According to the [doc](https://docs.buf.build/breaking/configuration#except) you can just write:
```suggestion
    - PACKAGE
  except:
    - ENUM_VALUE_NO_DELETE
    - FIELD_NO_DELETE
```